cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,23 @@
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
```

Comment:
Yes, it would be dangerous. From my understanding, the purpose of this is to **automatically** remove the installed files right after it's installed (assuming the above situation hasn't happened) instead of removing them manually something like:
```shell
sudo rm -rf /usr/local/include/aikido
sudo rm /usr/local/lib/libaikido*
...
```